src/config/yaml_baking_conf_parser.py

```diff
@@ -351,7 +351,7 @@ def validate_rewards_type(self, conf_obj):
                 "Defaults to 'actual' rewards payout type.".format(REWARDS_TYPE)
             )
 
-        if conf_obj[REWARDS_TYPE].isEstimated():
+        if conf_obj[REWARDS_TYPE] == RewardsType.ESTIMATED:
```

Comment:
This is strange that `conf_obj[REWARDS_TYPE]` can be a string which needs further investigations why this is happening. For now I just port back as a quick hotfix.

Reply:
Can't you test for a string and make a case handling here? Maybe there is no proper cast in die config loader?